app/models/contributor.rb

```diff
@@ -109,9 +108,7 @@ def channels
   end
 
   def active_request
```

Comment:
Maybe we can do away with this method altogether? `received_messages.first&.request` sounds a lot better than `active_request`.